backend/LeetWars.Core/LeetWars.Core.WebAPI/appsettings.json

```diff
@@ -12,5 +12,12 @@
       "Microsoft.AspNetCore": "Warning"
     }
   },
-  "AllowedHosts": "*"
+  "AllowedHosts": "*",
+  "RabbitURI": "amqp://guest:guest@localhost:5672",
```

Comment:
are you sure this will work without docker?

Reply:
so you mean app should be able to work both on docker and locally without changing RabbitURI?

Reply:
we have a URL for docker in the docker-compose file, so in docker we need to use it

Reply:
so you need to make it work on a local machine, on staging we will store URL in the env variables

Reply:
I was just little bit confused about your question. Yes, I tested it from test endpoint that i created, but I didn't include it in my commits. Everything works fine locally